projects/ngrx.io/content/examples/testing-store/src/app/state/books.reducer.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('BooksReducer', () => {
       const state = fromReducer.booksReducer(initialState, action);
 
       expect(state).toEqual(newState);
-      expect(state).not.toBe(newState);
+      expect(state).not.toEqual(initialState);
```

Comment:
We still want to use `toBe` to make sure they don't share the same reference.

```suggestion
      expect(state).not.toBe(initialState);
```